v2/pkg/core/inputs/hybrid/hmap.go

```diff
@@ -97,6 +98,13 @@ func (i *Input) initializeInputSources(options *types.Options) error {
 		i.scanInputFromReader(input)
 		input.Close()
 	}
+	if options.Uncover {
```

Comment:
this logic should be the part of a dedicated `uncover` subpackage

Reply:
moved this  to new package